.env

```diff
@@ -0,0 +1,8 @@
+acm_cert_arn
+AWS_ACCESS_KEY_ID
+AWS_REGION
+AWS_SECRET_ACCESS_KEY
+domain_name
```

Comment:
Not a big deal at all but having everything in all-caps would be a nice touch ☺️

Reply:
I agree. I have to test it to see if it is case sensitive or not. I will fix and open another PR. Thaks for your review!